### PR TITLE
(2.0.x) Fix for issue #7999

### PR DESCRIPTION
**Pull request by SCiunczyk**
Fix for LED/Neopixel compilation problems described in issue #7999. 

Added function leds->set_led_white();

Fixed error with Neopixel startup test. Too long delay was fired up  a processor watchdog. 
The first delay was unnecessary and 1000ms delay IMHO is enough to signalise that LED Strip is working. Cumulative delay is <4s

Polish translation of MSG_USER_MENU (I'm using User menu for LED On/Off function)

### CODE REVIEW

**Comment by Tannoo** [comment]
Did you also do this for 1.1.x?

File: Marlin/src/feature/leds/neopixel.cpp [hunk 1]
```diff
@@ -44,15 +44,14 @@ void setup_neopixel() {
   pixels.show(); // initialize to all off
 
   #if ENABLED(NEOPIXEL_STARTUP_TEST)
-    delay(2000);
     set_neopixel_color(pixels.Color(255, 0, 0, 0));  // red
-    delay(2000);
+    delay(1000);
     set_neopixel_color(pixels.Color(0, 255, 0, 0));  // green
```
Comment (AnHardt) [inline Marlin/src/feature/leds/neopixel.cpp]: Even if the delays are much shorter now - safe_delay() is the function made for delays longer than a few ms.

**Comment by SCiunczyk** [comment]
Thanks @AnHardt. It's done.

**Comment by SCiunczyk** [comment]
> Did you also do this for 1.1.x?

no, I can't compile bugfix-1.1.x. I got errors with TMC26XStepper...

**Comment by thinkyhead** [comment]
Changes applied to 2.0.x to match the work in 1.1.x. The addition of `set_led_white` isn't needed since `set_led_color` now manages recognizing white.

Thanks for the contrib! Polish strings have also been added.

**Comment by SCiunczyk** [comment]
> The addition of set_led_white isn't needed since set_led_color now manages recognizing white.

Yes and not ;-) For example with `set_led_white` function you can have the M104_M109.cpp clean of this construction: 
```
#if ENABLED(RGB_LED) || ENABLED(BLINKM) || ENABLED(PCA9632) || ENABLED(RGBW_LED)
  set_led_color(LED_WHITE);
#endif
#if ENABLED(NEOPIXEL_LED)
  set_neopixel_color(pixels.Color(NEO_WHITE));
#endif
```

**Pull request closed without merging**